roles/draupnir/tasks/config.yml

```
@@ -0,0 +1,24 @@
+---
+
+- name: Check if data dir exists
+  ansible.builtin.stat:
+    path: "{{ draupnir_data_path }}/data"
+  register: draupnir_dir_stat
+
+- name: Create directory if it does not exist
+  ansible.builtin.file:
+    path: "{{ draupnir_data_path }}/data"
+    state: directory
+    owner: draupnir
+    group: draupnir
+    mode: '0755'
+  when: not draupnir_dir_stat.stat.exists
```

Miksi tässä on stat ja file, eikä vaan

```suggestion
- name: Create directory if it does not exist
  ansible.builtin.file:
    path: "{{ draupnir_data_path }}/data"
    state: directory
    owner: draupnir
    group: draupnir
    mode: '0755'
```

file-moduulin pitäis olla ihan idempotentti itsessään.

---

edelleen relevantti